<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcquisitionWidgetVerticalLayout</class>
 <widget class="QWidget" name="AcquisitionWidgetVerticalLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>291</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Acquisition widget</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="acq_gbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>32767</width>
       <height>32767</height>
      </size>
     </property>
     <property name="title">
      <string>Acquisition</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="osc_start_label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1677215</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Oscillation start:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="first_iamge_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>First image:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="exp_time_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Exposure time (s):</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="kappa_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Kappa:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="energy_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Energy (keV):</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="resolution_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Resolution (Ã…):</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="transmission_label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Transmission (%):</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="shutterless_cbx">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Shutterless</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="inverse_beam_cbx">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Check to use inverse beam and specify subwedge size</string>
              </property>
              <property name="text">
               <string>Inverse beam</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLineEdit" name="osc_start_ledit">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="first_image_ledit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="exp_time_ledit">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="kappa_ledit">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="energy_ledit">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="resolution_ledit">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="transmission_ledit">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>56</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="osc_range_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Oscillation range:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="num_images_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>27</height>
               </size>
              </property>
              <property name="acceptDrops">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Number of images:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="detector_roi_mode_label">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>160</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Detector mode:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="kappa_phi_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Phi:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="mad_cbox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>27</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Check to use energies from energy scan</string>
              </property>
              <property name="text">
               <string>MAD</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="subwedge_size_label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>27</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Check to use inverse beam and specify subwedge size</string>
              </property>
              <property name="text">
               <string>Subwedge size:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLineEdit" name="osc_range_ledit">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="num_images_ledit">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="detector_roi_mode_combo">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Select necesaary detector mode</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="kappa_phi_ledit">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="energies_combo">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Check to use energies from energy scan</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="subwedge_size_ledit">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Check to use inverse beam and specify subwedge size</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
